### PR TITLE
Updated README.md with global system related dependencies

### PR DESCRIPTION
**Pull request by nunoguedelha**
The dependencies are related to the options `ROBOTOLOGY_ENABLE_ICUB_HEAD` and `ROBOTOLOGY_ENABLE_DYNAMICS`.

### CODE REVIEW

**Comment by traversaro** [comment]
Thanks, I guess the problems that you mentioned f2f with the `libdc1384` homebrew formula has been fixed?

**Comment by nunoguedelha** [comment]
Not really, I had to put on hold the analysis for proceeding with my initial goal. I need to take a deeper look tomorrow. I will document it if it turns out to be a real issue.

**Comment by traversaro** [comment]
Then I would wait before documenting the installation of `libdc1384` , otherwise we may be suggesting the users to install something that is breaking their builds. : )

**Comment by nunoguedelha** [comment]
Yes sure, we don't merge this until it is clarified.

**Comment by traversaro** [comment]
Given https://github.com/robotology/robotology-superbuild/pull/274 , I guess we can now close this PR?

File: README.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ user@host:~$ sudo ldconfig
 ### System Dependencies
 To install the system dependencies, it is possible to use [Homebrew](http://brew.sh/):
 ```
-brew install ace eigen cmake boost tinyxml swig qt5 gsl pkg-config jpeg sqlite readline tinyxml ipopt opencv@3
+brew install ace eigen cmake boost tinyxml swig qt5 gsl pkg-config jpeg sqlite readline tinyxml ipopt opencv@3 libdc1384
 ```
```
Comment (nunoguedelha) [inline README.md]: This change can indeed be dropped for now.

File: README.md [hunk 2]
```diff
@@ -481,7 +481,15 @@ Install matio using the following command:
 sudo apt install libmatio-dev
 ```
 
-#### Windows and macOS
+#### macOS
+
+Install matio from `homebrew/core` using the following command:
+
+```
+brew install libmatio
```
Comment (nunoguedelha) [inline README.md]: but we still want this one.

Comment (nunoguedelha) [reply README.md]: CC @traversaro

Comment (traversaro) [reply README.md]: Ack.

**Comment by nunoguedelha** [comment]
Rebased.

**Pull request merged**